index.html

```diff
@@ -74,5 +74,105 @@ <h3>Skills</h3>
         </div>
     </div>
 
+    <h1 class="workHeadingMain">MY RECENT WORK</h1>
+    
+    <div class="workGridContainer">
+        <div class="secondpage">
+            <div class="horizontalline">
+
+            </div>        
+            <div class="workimage">
+                <img src="ImgPlaceholder.png" alt="Project Image">
+            </div>
+            <h2 class="workheading">Multi-Post Stories</h2>
+            <p class="worktext">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <div class="sidebar">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
+            <button type="button" class="projectButton">See project</button>
+        </div>
+
+        <div class="secondpage">      
+            <div class="workimage">
+                <img src="ImgPlaceholder.png" alt="Project Image">
+            </div>
+            <h2 class="workheading">Multi-Post Stories</h2>
+            <p class="worktext">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+            <div class="rightsidebar">
+
+            </div>
+            
+            <div class="languagesFlexContainer">
+                <span class="languages">CSS</span>
+                <span class="languages">HTML</span>
+                <span class="languages">Bootstrap</span>
+                <span class="languagestwo">Ruby</span>
+            </div>
```

Comment:
- [ ] The list of the project's technologies is a list `ul` and each of them a `li`, please kindly fix it 😁